sidecars/python/etc/entrypoint.sh

```diff
@@ -28,4 +28,13 @@ if [ "${USER_ID}" -ne 0 ] && command -v sudo >/dev/null 2>&1 && sudo -n true > /
     sudo chown "${USER_ID}:${GROUP_ID}" /projects
 fi
 
+# Setup .venv in the 'venv' volume that should be mounted in HOME/.venv
+cd "${HOME}"
+python -m venv .venv
+# shellcheck disable=SC1091
+. .venv/bin/activate
```

Comment:
Convention is usually to use `source`, we should probably install bash in the sidecar container for this reason

Reply:
any sources of this convention?

Reply:
Yes, here: https://docs.python.org/3/tutorial/venv.html#creating-virtual-environments

Reply:
ok let's install bash

Reply:
actually bash is already installed!